src/legend_detector_to_siggen.jl

```diff
@@ -87,13 +87,14 @@ function siggen_config(meta::PropDict, env::Environment, simulator::SiggenSimula
     # add to field fieldgen_names
     fieldgen_names["drift_name"] = cache_drift_file
 
-    # why is this needed? 
-    for name in ["field_name", "wp_name"]
-        path = joinpath("cache", fieldgen_names[name])
-        if !isfile(path)
-            touch(path)
-        end
-    end
+    ## commenting this out: produced a bug
+    ## (empty file created, next time running empty file read and no field at point error)
+    # for name in ["field_name", "wp_name"]
+    #     path = joinpath("cache", fieldgen_names[name])
+    #     if !isfile(path)
+    #         touch(path)
+    #     end
+    # end
```

Comment:
What's this about? 
Is this not needed or just to avoid a bug?

Reply:
This piece of code created empty siggen field and wp output files. If then nothing was saved there because of some error, on the next run, instead of running the detector simulation from scratch, LegendGeSim would detect that cached files exist, and read them out. There wouldn't be any error, but the fields would be empty, and later with PSS siggen would say "no field at point".

Removing this part, the bug was fixed, and I didn't see anything going wrong without it. I don't know why it would be needed to create empty files, they are written later without having the need to have empty ones first. These lines were added by @lmh91 I believe, so I don't know their purpose.